samples/nrf9160/coap_client/README.rst

```diff
@@ -3,12 +3,24 @@
 nRF9160: nRF CoAP Client
 ########################
 
-The nRF CoAP Client sample demonstrates how to receive data from a public CoAP server with an nRF9160 SiP.
+The nRF CoAP Client sample demonstrates the communication between a public CoAP server and an nRF9160 SiP which acts as the CoAP client.
 
 Overview
-*********
+********
 
-The sample connects to a public CoAP test server, sends periodic GET request for a test resource that is available on the server, and prints the data that is received.
+The nRF CoAP Client sample performs the following actions:
+
+#. Connect to the configured public CoAP test server (indicated by the parameter ``CONFIG_COAP_SERVER_HOSTNAME``)
+#. Send periodic GET request for a test resource (indicated by the parameter ``CONFIG_COAP_RESOURCE``) that is available on the server
+#. Display the received data about the resource on a terminal emulator.
+
+The public CoAP server used in this sample is Californium CoAP server (coap://californium.eclipse.org:5683).
+This server runs Eclipse Californium which is an open source implementation of the CoAP protocol that is targeted at the development and testing of IoT applications.
+An nRF9160 DK board is used as the CoAP client.
+	
+An observable resource **obs** (Californium observable resource) is used in the communication between the CoAP client and the public CoAP server.
+The communication follows the common observe pattern and it defines two roles - an observer and an observable subject.
+The CoAP client or the observer, is notified of any changes in the state of the value of the observable resource from the server.
```

Comment:
Is Nordic's smartcoAP library  used in the implementation?

Reply:
No, Nordic implementation is not available in NCS.